docs/source/sysadmin_guide/configuring_sawtooth/validator_configuration_file.rst

```diff
@@ -191,3 +191,12 @@ The ``validator.toml`` configuration file has the following options:
     ``sawtooth identity`` CLI to change this on-chain-only setting.
 
   See :doc:`../permissioning` for more information on roles and permissions.
+
+
+  The minimum number of peers required before stopping peer search.
```

Comment:
The bullet & option name are missing - these are just descriptions. 

Tip: Copy a simple entry (like "scheduler" and use the same format:

- Start with a bullet and the option syntax in this format:
     \- \`\`optionname\`\` = \`value\`
     Replace "value" with a unique word for each (I suggest min and max)
- Include the default, such as "Default: 3.".  Say "Default: none." if there's no default.